volatility3/framework/plugins/windows/malfind.py

```diff
@@ -112,13 +112,13 @@ def list_injection_sites(
         code.
 
         Args:
-            context: The context to retrieve required elements (layers, symbol tables) from
+            context: The context to retrieve required elements (layers, symbol tables)
```

Comment:
```suggestion
            context: The context from which to retrieve required elements (layers, symbol tables)
```
Retrieve requires a location, but ending in a preposition isn't great English, so technically this include "from which" somewhere earlier in the sentence.